help/md/fnJC.md

```diff
@@ -3,15 +3,15 @@ fnJC
 ## title
 fnJC
 ## description
-Jukes cantor transition matrix
+Jukes Cantor rate matrix
 ## details
-Transition matrix with n amount of states in which each state has an equal
-probability of to change to any other state. The only parameter in this model
-is mu with the rate of change from any given state being mu / number of states.
+Rate matrix with n states in which each state has an equal probability of to change 
+to any other state. The rate of transition from one state to another is equal to 
+n / n-1.
 ## authors
 Jukes & Cantor 
 ## see_also
-tbd ~ other mutation models?
+fnGTR, F81
```

Comment:
"F81" should read fnF81, and it might be cleaner to separate items by line breaks rather than commas

Reply:
Yes, we should use separate lines, at least for now, because the website machinery can't handle commas (yet) when it makes links.